plugin/src/com/microsoft/alm/plugin/external/utils/ProcessHelper.java

```diff
@@ -27,6 +27,6 @@ public static Process startProcess(final String workingDirectory, final List<Str
     private static String getPatchedPathWithCurrentJavaBinLocation() {
         String currentJavaHome = System.getProperty("java.home");
         String originPathVariable = System.getenv("PATH");
-        return originPathVariable + File.pathSeparator + currentJavaHome + File.separator + "bin" + File.separator;
+        return File.pathSeparator + currentJavaHome + File.separator + "bin" + File.separator + File.pathSeparator + originPathVariable;
```

Comment:
Why does this need to start with File.pathSeparator?

Reply:
@jeschu1 Fixed.